action.yml

```diff
@@ -90,5 +90,5 @@ outputs:
   image-with-tag:
     description: 'Name of the image tagged with the first tag present'
 runs:
-  using: 'node12'
+  using: 'node16'
   main: 'dist/index.js'
```

Comment:
Do you need to regenerate this file?

Reply:
I did with `podman run --rm -it -v $(pwd):/project --entrypoint=/bin/bash -w /project node:16 -c "npm ci && npm run bundle"`, but there were no changes.

Reply:
Thank you